terraform/modules/filecoin_node/lotus.sh

```diff
@@ -51,6 +51,13 @@ fi
 # Change the ownership of the lotus_data directory to the created user.
 chown --recursive "${NEW_USER}":"${NEW_USER}" "/home/${NEW_USER}/lotus_data"
 
+cat << EOF > "/home/${NEW_USER}/lotus_data/config.toml"
+[Chainstore]
+  EnableSplitstore = true
+  [Chainstore.Splitstore]
+    ColdStoreType = "discard"
```

Comment:
```suggestion
[Chainstore]
EnableSplitstore = true

[Chainstore.Splitstore]
ColdStoreType = "discard"
```

Weird indentation

Reply:
@samuelarogbonlo Indentation (or empty lines) does not affect the logic in TOML. TOML ain't YAML. :)